src/workflow-engine/README.md

```diff
@@ -82,3 +82,61 @@ microk8s/3 -- microk8s kubectl get nodes`. You'll see something like:
 
 - the cluster nodes should also be accessible via SSH: `juju ssh microk8s/3` or
 `ssh ubuntu@<ip>`.
+
+### Deploy a dedicated VM for the microk8s' Juju Controller
+
+In order to deploy Kubernetes charms to the above microk8s cluster, we'll need
+to bootstrap a Juju controller for that cloud.
+In theory, we could use PS6 Juju client for this, but it is better not to
+pollute that environment. So let's create a dedicated VM,
+`rocks-ps6-juju-client`:
+
+```bash
+# From inside the PS6 environment, with the openstack credentials loaded
+openstack keypair list
+# Is there isn't a "rocks-ps6-keypair" entry, create one with:
```

Comment:
You wanted to write "if there isn't" no ?

Reply:
True :) thanks

Reply:
https://github.com/canonical/oci-factory/pull/126/commits/c919524c1f82370712482dcd2a204d82198d1069